Automattic/sensei
# Add Lists and Subscribers

Fixes #6426 

### Changes proposed in this Pull Request

* Adds a MailPoet list for every course and student group.
* For each enrolled student in every course or group, adds subscribers to each list.

### Testing instructions

- Have groups and courses on your site.
- Add students to each group and course for testing MailPoet integration.
- Install MailPoet plugin.
- Go through the setup wizard, sign up for MailPoet and add API key to your plugin installation if you don't have an account.
- Visit MailPoet > Lists: 
  - Check to see that new lists are created for each course and group in your site.
- Visit MailPoet > Subscribers:
  - Make sure that all enrolled students in each course are subscribers to the course list on MailPoet. Check the same for each group.

### Screenshot / Video
